samples/compose/src/main/java/com/squareup/sample/compose/hellocomposebinding/HelloBindingActivity.kt

```diff
@@ -0,0 +1,53 @@
+package com.squareup.sample.compose.hellocomposebinding
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.material.MaterialTheme
+import androidx.lifecycle.SavedStateHandle
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.squareup.workflow1.ui.ViewEnvironment
+import com.squareup.workflow1.ui.ViewRegistry
+import com.squareup.workflow1.ui.WorkflowLayout
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.compose.withCompositionRoot
+import com.squareup.workflow1.ui.renderWorkflowIn
+import kotlinx.coroutines.flow.StateFlow
+
+@OptIn(WorkflowUiExperimentalApi::class)
+private val viewRegistry = ViewRegistry(HelloBinding)
+
+@OptIn(WorkflowUiExperimentalApi::class)
+private val viewEnvironment =
+  ViewEnvironment(mapOf(ViewRegistry to viewRegistry)).withCompositionRoot { content ->
+    MaterialTheme(content = content)
+  }
+
+class HelloBindingActivity : AppCompatActivity() {
```

Comment:
A quick KDoc explaining what this activity is demonstrating – or documentation wherever it fits best – would be a win for busy folks who come here later and don't want to dive into the code. 

(Same for `SampleLauncherApp()`.)